### PR TITLE
README update

### PR DESCRIPTION
**Pull request by nhakmiller**
### Background

Updated the README to more accurately reflect the dual nature of the repository.

### Changes

* Updated the README structure to discuss `analysis` and `panther-cli` separately
* Added Austin as a code owner

### Testing

* N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,38 @@
-# Panther CLI Tool
-`panther-cli` is a Python command line interface for testing and packaging Panther Policies and Rules. Policies define the compliant and secure state of a cloud Resource, whereas Rules perform analysis on log data.
+# Panther Analysis
 
-## Installation
+This repository is a combination of open source policies and rules meant to be run with [Panther](https://github.com/panther-labs/panther), as well as the `panther-cli` tool which can be used to test and package these policies and rules without having to go through the Panther web interface.
```
Comment (austinbyers) [inline README.md]: Can we mention something about the artifacts of the releases in this repo are bundled in Panther deployments by default?

**Comment by austinbyers** [comment]
Can you add me as a CODEOWNER as well?

File: README.md (hunk 1 above)
Comment (nhakmiller) [reply README.md]: Will do, added a brief paragraph explaining the relationship.

**Comment by nhakmiller** [comment]
> Can you add me as a CODEOWNER as well?

I suppose if I am going to keep tagging you for review I probably should :)

File: README.md (hunk 1 above)
Comment (jacknagz) [reply README.md]: This repository contains open-source Panther policies and rules, along with a CLI tool for testing and packaging.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,44 @@
...
+
+During initial deployments of Panther, the default configuration is to load all the policies and rules published during version releases of this project. This provides a set of out-of-the-box policies and rules to get analysis up and running. See the Panther documentation for how to override this default behavior if desired.
```
Comment (jacknagz) [inline README.md]: During the initial deployment of Panther, all of the policies and rules published here are uploaded. This provides a set of out-of-the-box policies and rules to establish a strong detection baseline. See the [Panther documentation](https://docs.runpanther.io) for how to override this default behavior if desired.

**Pull request merged**